### ISSUE TITLE
[suggestion] smaller continent zones

### ISSUE THREAD

**Issue by SuspiciousMango** [BlackJar72/ClimaticBiomePlacement#34]
First of all, I'd just like to say that I have been using this mod for world gen in my modpacks for about a year now and wow, its everything I've ever wanted from a world gen mod. I remember seeing the original 1.0.0 version pop up on forge and I couldn't wait until some of the bugs got fixed and some of the settings got tweaked so I could use it, and I have been using it ever since.

The only issue I have with it (and I'm sure you can guess by the title of this issue) is that I find 4096x4096 to just be WAY too big. I really like the idea of continents to explore, but I never find myself exploring them because, even on a flying mount like an ice and fire dragon, a single continent is just so massive that there's never really a need to go anywhere else. I would love it if there was a config option to change the size of the continent zones (maybe in terms of chunks instead of by blocks because I feel like the player setting the number to a non-chunk aligned number might break stuff). As someone who knows many of the ins and outs of java and knows absolutely nothing about minecraft modding, I have no idea if this is even possible, but its something I'd really love to see from one of my favorite mods out there.

TL:DR - Smaller continent zones would be amazing, but I have absolutely no idea if they're possible or not and totally understand it if they aren't.

**Comment by BlackJar72** [comment]
It could be done.  It's the size scale, which I at first added because some players wanted huge continents; so far it doesn't do smaller, just integer multiples of 4096.  As an alternative I suppose I could add a 1/2 setting, I'm just not sure how it would look in the config ("0" would be misleading as everything else is a multiplier).  But 1/2 would give the same size as a level 5 map, which is what I often treated as my world for a long time.  So in theory it could be done; I'll look into it.

It plans the continents on the scale of chunks, so it will always be even chunks across.  I'd probably also avoid dividing by anything put a power of two.  Then, a tiny continent wouldn't make that much since, so less then 2048x2048 is probably as small as it needs to go.

I'd actually rather do little things like this that anything big right now.

**Comment by BlackJar72** [comment]
I like the idea, though its turning out to be hard than I expected.

**Comment by BlackJar72** [comment]
I still like the idea, but it might be harder than I thought.
